QtCollider/widgets/QcWebView.cpp

```diff
@@ -159,7 +158,8 @@ void WebView::toPlainText(QcCallback* cb) const {
 void WebView::runJavaScript(const QString& script, QcCallback* cb) {
     if (page()) {
         if (cb) {
-            page()->runJavaScript(script, cb->asFunctor());
+            // convert QVariant to string until we deal with QVariants
+            page()->runJavaScript(script, [cb](const QVariant& t) { cb->call(t.toString()); });
```

Comment:
i don't understand this comment

Reply:
thanks for looking into it! since this is the last PR before the 3.11.2 release, let's just leave this as it is. if you're interested you can make another PR for it, i'd also be happy to take a look myself.

Reply:
ok, we keep it like it is then and I'll open another PR! Just to be extra clear, now all javascript calls from sclang will always return a string only